src/main/java/com/pusher/client/channel/impl/PresenceChannelImpl.java

```diff
@@ -26,6 +28,10 @@
 
     private final Map<String, User> idToUserMap = Collections.synchronizedMap(new LinkedHashMap<String, User>());
 
+    private static final Gson gson = new GsonBuilder()
```

Comment:
Constants should be presented about final members (`idToUserMap`) and should be named in ALL_CAPS.